src/Type/Object/TermObject.php

```diff
@@ -62,13 +66,28 @@ function register_taxonomy_object_type( $taxonomy_object ) {
 		register_graphql_field( $taxonomy_object->graphql_single_name, 'parent', [
 			'type'        => $taxonomy_object->graphql_single_name,
 			'description' => __( 'The parent object', 'wp-graphql' ),
+			'resolve' => function( Term $term, $args, $context, $info ) {
+				return isset( $term->parentId ) ? DataSource::resolve_term_object( $term->parentId, $context ) : null;
+			}
 		] );
 
 		register_graphql_field( $taxonomy_object->graphql_single_name, 'ancestors', [
 			'type'        => [
 				'list_of' => $taxonomy_object->graphql_single_name,
 			],
 			'description' => esc_html__( 'The ancestors of the object', 'wp-graphql' ),
+			'resolve' => function( Term $term, $args, $context, $info ) {
+				$ancestors    = [];
+
+				$ancestor_ids = get_ancestors( absint( $term->term_id ), $term->taxonomyName, 'taxonomy' );
+				if ( ! empty( $ancestor_ids ) ) {
+					foreach ( $ancestor_ids as $ancestor_id ) {
+						$ancestors[] = DataSource::resolve_term_object( $ancestor_id, $context );
+					}
+				}
+
+				return ! empty( $ancestors ) ? $ancestors : null;
+			}
 		] );
 
 		// @todo
```

Comment:
Do we have an issue for this? If so, we should reference it in the comment

Reply:
ha! That was resolved a while (few months) ago, just never cleaned up the todo.